examples/test/GeoPushpinChart_test.js

```diff
@@ -0,0 +1,61 @@
+// (C) 2020 GoodData Corporation
+import { Selector } from "testcafe";
+import { config } from "./utils/config";
+import { loginUsingLoginForm } from "./utils/helpers";
+import { DEFAULT_COLOR_PALETTE } from "../src/utils/colors";
+
+fixture("Geo Pushpin Chart Demo") // eslint-disable-line no-undef
```

Comment:
can we remove ` // eslint-disable-line no-undef`?

Reply:
fixed